llvm/test/tools/llvm-symbolizer/skip-line-zero.s

```diff
@@ -0,0 +1,402 @@
+##  Test  --skip-line-zero option.
+##
+##  This test illustrates the usage of handcrafted assembly to produce the following line table.
+##  Address            Line   Column File   ISA Discriminator OpIndex Flags
+##  ------------------ ------ ------ ------ --- ------------- ------- -------------
+##  0x0000000000001710      1      0      2   0             0       0  is_stmt
```

Comment:
I think that @dwblaikie wanted the is_stmt/epilogue_begin/column instructions removed? It would probably be best to remove everything that is not important to --skip-line-zero from this table.

Reply:
Yeah, mixed feelings - if it's literally hand-crafted assembly, I'd advocate for simplifying it as much as possible (removing stmt, epilogue, column, etc seems good)

If it's /mostly/ generated assembly with a couple of choice edits, simplifying the process to create it may be sufficient - like maybe turning of column info when running clang to generate it. (maybe optimizing the code, which would probably remove the prologue/epilogue? Not sure there's much we can do from the source level about is_stmt... )

If the hand-crafted assembly can be made short enough to be fairly readable/writable directly (with comments) - that's probably good.

(& sorry for all the back and forth on this... :/ )

Reply:
Done.